From 377a5a9a81a3063ab56e2916575d46c0ec984f52 Mon Sep 17 00:00:00 2001
From: linxiaoxu131415 <linxiaoxu@huawei.com>
Date: Wed, 13 Oct 2021 07:54:55 +0000
Subject: [PATCH] =?UTF-8?q?=E4=BF=AE=E6=94=B9=E5=8F=82=E6=95=B0=E6=A0=A1?=
 =?UTF-8?q?=E9=AA=8C=E8=8C=83=E5=9B=B4?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

---
 scripts/generate.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/generate.sh b/scripts/generate.sh
index c181221..7a57b05 100644
--- a/scripts/generate.sh
+++ b/scripts/generate.sh
@@ -291,7 +291,7 @@ if [ $# != 4 ];then
 fi
 
 set +eE
-for i in $*
+for i in $1 $2 $3
 do
    echo "$i" | grep -v -E ${CHECK_REGEX}
    filterParam=$(echo "$i" | grep -v -E ${CHECK_REGEX})
-- 
2.23.0

